.github/workflows/e2e.yml

```diff
@@ -21,3 +21,5 @@ jobs:
         env:
           SITECORE_CH1_ENDPOINT_URL: ${{ secrets.SITECORE_CH1_ENDPOINT_URL }}
           SITECORE_CH1_CLIENT_KEY: ${{ secrets.SITECORE_CH1_CLIENT_KEY }}
+          AZURE_TABLE_NAME: 'Urls'
+          AZURE_TABLE_CONNECTION_STRING: ${{ secrets.AZURE_TABLE_CONNECTION_STRING }}
```

Comment:
Yes, it does mean anyone (including the tests) could create ids to json objects.  But I'm not sure what difference it'll make if that object is created in a non production environment vs production.  If that Json object is never matched up to again, it'll never be used again.  If a row is created and it is attempted to be used again because the json object matches, does it really matter if it uses that ID? I would think those cases improve performance in fact.

We could consider a dev environment, but with what is mentioned above, I'm not sure its necessary, plus this would come with additional costs.

Reply:
Okay, i'm fine tabling this until it becomes a problem, if ever.